README.md

```diff
@@ -1,9 +1,66 @@
-# triton-docker
+# Triton Docker CLI helper
 
-Docker CLI helper for Triton
+This script installs known-good, tested versions of the Docker (now Moby) and Docker Compose CLI tools for use with Triton.
```

Comment:
> "known good" should be hyphen free

Thanks for putting me straight

> Maybe link out now Moby to the related PR release

I'd honestly prefer not validating that whole thing with a link. It was and remains a fiasco that people are hugely confused about.

> as the name of the project updates

I would likely not change the name `triton-docker`, even if Docker does more name changing.